src/main/java/com/todolist/app/servlet/EditTaskServlet.java

```diff
@@ -0,0 +1,57 @@
+package com.todolist.app.servlet;
+
+import com.todolist.app.model.*;
+import com.todolist.app.service.ToDoStoreSingleton;
+import jakarta.servlet.*;
+import jakarta.servlet.http.*;
+import jakarta.servlet.annotation.*;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@WebServlet(name = "EditTaskServlet", value = "/edit-task")
+public class EditTaskServlet extends HttpServlet {
+
+    ToDoStoreSingleton singleton = ToDoStoreSingleton.getInstance();
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
By the way, can we implement a test on this? E.g. https://stackoverflow.com/questions/5434419/how-to-test-my-servlet-using-junit